README.md

```diff
@@ -21,3 +21,26 @@ A fragment may be reasonable to insert into multiple locations. However, downstr
 ## Files produced
 
 The plugin will generate two files, one which is a `Phylogeny[Rooted]` type, and once which is a `Placements` type. The former is the tree with the sequences placed (which could be inserted), and are identified by their corresponding sequence IDs. The latter is a JSON object which, for every input sequence, describes the different possible placements. 
+
+## Example
+
+QIIME 2's "Moving Pictures" [tutorial](https://docs.qiime2.org/2017.10/tutorials/moving-pictures/#generate-a-tree-for-phylogenetic-diversity-analyses) suggests constructing a de-novo phylogeny for the fragments, i.e `FeatureData[Sequence]`, to obtain a `Phylogeny[Rooted]` that can be used for phylogenetic diversity computation. "Fragment insertion" via this plugin provides an alternative way to acquire the `Phylogeny[Rooted]` by inserting sequences of `FeatureData[Sequence]` into a high quality reference phylogeny and thus provides multiple advantages over de-novo phylogenies, e.g. accurate branch lengths, multi-study meta-analyses, mixed region meta-analyses (e.g. V4 and V2).
+
+Let us use the `FeatureData[Sequence]` from QIIME's tutorial as our input:
+
+   - `rep-seqs.qza`: [view](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2017.10%2Fdata%2Ftutorials%2Fmoving-pictures%2Frep-seqs.qza) | [download](https://docs.qiime2.org/2017.10/data/tutorials/moving-pictures/rep-seqs.qza)
+
+The following single command will produce three outputs: 1) `phylogeny.qza` is the `Phylogeny[Rooted]`, 2) `placements.qza` provides placement distributions for the fragments (you will most likely ignore this output) and 3) `classification.qza` which is a taxonomic classification for every fragment that has been inserted into the reference phylogeny and is of the type `FeatureData[Taxonomy]` (Computation might take some 10 minutes):
+```
+qiime fragment-insertion sepp-16s-greengenes \
+  --i-representative-sequences rep-seqs.qzv \
+  --o-tree insertion-tree.qza \
+  --o-placements insertion-placements.qza \
+  --o-classification insertion-taxonomy.qza
+```
+Output artifacts:
+   - `insertion-tree.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/raw/readme_example/Example/insertion-tree.qza)
+   - `insertion-placements.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/raw/readme_example/Example/insertion-placements.qza)
+   - `insertion-taxonomy.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/raw/readme_example/Example/insertion-taxonomy.qza)
```

Comment:
why are the view links crossed out?

Reply:
because I don't know how to set them up correctly :-/

Reply:
I keep getting an `TypeError: NetworkError when attempting to fetch resource.` :-(

Reply:
Strange. In Chrome the same link lead to a different error message: `TypeError: Failed to fetch`